src/Api/Controllers/OrganizationsController.cs

```diff
@@ -143,6 +148,26 @@ public async Task<ListResponseModel<ProfileOrganizationResponseModel>> GetUser()
             var responses = organizations.Select(o => new ProfileOrganizationResponseModel(o));
             return new ListResponseModel<ProfileOrganizationResponseModel>(responses);
         }
+        
+        [HttpGet("{identifier}/auto-enroll-status")]
```

Comment:
Auto enroll feels a little vague here.

Is the intent to eventually have this endpoint give information about any organization auto-enroll requirements?

Reply:
Not sure I follow - have suggestions?

Reply:
So this is for auto enroll to APR. Will there be other is the intent to add other auto enrolls to this someday? maybe I'm just seeing premature changes that we could do later when needed

Reply:
Ahh - unsure about the future. This API is very purpose built to return specifically what's needed during the JIT SSO user `set-password` flow. It could be made more generic to just return an organization's ID and "other" policy enabled states?

Reply:
OK I think the vagueness of the name will end up being a boon in the future.

That being said, I think it's worth being a little future-looking and rename the property in the model to be specific to the policy feature. I'm thinking something like `adminPasswordResetEnabled`. That way if something is added to this model we won't have to worry about old clients having a too-generic name.

Reply:
Understood. Updated!